enterprise/api/v1/views.py

```diff
@@ -843,12 +844,49 @@ def _get_return_status(self, serializer, many):
         return status.HTTP_204_NO_CONTENT
 
     def create(self, request, *args, **kwargs):
+        """
+        If a Django user exists, but the correspinding EnterpriseUser does not, creates a PendingEnterpriseUser for
+        the email in question.
+        Can accept one user or a list of users.
+
+        Returns 201 if any users were created, 204 if no users were created.
+        """
         serializer = self.get_serializer(data=request.data, many=isinstance(request.data, list))
         return_status = self._get_return_status(serializer, many=isinstance(request.data, list))
         headers = self.get_success_headers(serializer.data)
         return Response(serializer.data, status=return_status, headers=headers)
 
 
+class PendingEnterpriseCustomerUserEnterpriseAdminViewSet(PendingEnterpriseCustomerUserViewSet):
+    """
+    Viewset for allowing enterpise admins to create linked learners
+    Endpoint url: link_pending_enterprise_users/(?P<enterprise_uuid>[A-Za-z0-9-]+)/?$
+    Admin must be an administrator for the enterprise in question
+    """
+    permission_classes = (permissions.IsAuthenticated,)
+    serializer = serializers.LinkLearnersSerializer
+
+    @action(methods=['post'], detail=False)
+    @permission_required('enterprise.can_access_admin_dashboard', fn=lambda request, enterprise_uuid: enterprise_uuid)
+    def link_learners(self, request, enterprise_uuid):
+        """
+        If a Django user exists, but the correspinding EnterpriseUser does not, creates a PendingEnterpriseUser for
```

Comment:
I have to ask, why does a Django user have to exist? I did not think a Django user had to exist for a pending enterprise customer to be created if we did it from Django admin?

Reply:
Right. But if the EnterpriseUser already exists, we don't create a new PendingEnterpriseUser for that Django user.

How about:
Creates a PendingEnterpriseUser if no User exists or if the User exists but no EnterpriseUser exists
Does not create a PendingEnterpriseUser if the EnterpriseUser already exists.

Hmmm... I don't love that either; suggestions welcome.

Reply:
"Creates a `PendingEnterpriseCustomerUser` if no `EnterpriseCustomerUser` for the given (customer, email) combination(s) exists."

Reply:
Yeah, that seems accurate, thanks!